로또-최고순위-최저순위.js

```diff
@@ -0,0 +1,28 @@
+function solution(lottos, win_nums) {
+    
+    const answer = [];
+    let num = 0;
+    let countzero = 0;
+    let countmin = 0;
+
+    for (let i = 0; i <= 5; i++){
+        num = lottos[i];// lottos의 요소를 num에 넣는다. 
+
+        if (win_nums.indexOf(num) != -1) { //win_nums의 요소와 num이 동일하면
```

Comment:
==(동등 연산자) : 변수의 값 비교
===(일치 연산자) : 값 & 자료형 비교
앞으로는 엄격한 비교인 ===으로 해줄 것!
*모른다면 공부할 것*
```suggestion
        if (win_nums.indexOf(num) !== -1) { 
```